pkg/nsx/services/vpc/store.go

```diff
@@ -85,10 +89,18 @@ func (vs *VPCStore) GetVPCsByNamespace(ns string) []*model.Vpc {
 	for _, vpc := range vpcs {
 		mvpc := vpc.(*model.Vpc)
 		tags := mvpc.Tags
+		nameMatch := false
+		idMatch := false
 		for _, tag := range tags {
-			if *tag.Scope == common.TagScopeNamespace && *tag.Tag == ns {
-				ret = append(ret, mvpc)
+			if *tag.Scope == common.TagScopeNamespace && *tag.Tag == ns.Name {
+				nameMatch = true
 			}
+			if *tag.Scope == common.TagScopeNamespaceUID && *tag.Tag == string(ns.UID) {
```

Comment:
how about merge Line95/98? or if you think it's too long. you could check nameMatch first in line 98, if it's false, no need to compare the UID

Reply:
A tag`s Scope can not be both `common.TagScopeNamespaceUID` and `common.TagScopeNamespace`, I don’t think it can be merged here. Do you have a change suggestion?